src/main/java/com/radixdlt/crypto/hdwallet/bitcoinjprovider/HDWalletProviderBitcoinJ.java

```diff
@@ -0,0 +1,134 @@
+/*
+ *
+ *  * (C) Copyright 2020 Radix DLT Ltd
+ *  *
+ *  * Radix DLT Ltd licenses this file to you under the Apache License,
+ *  * Version 2.0 (the "License"); you may not use this file except in
+ *  * compliance with the License.  You may obtain a copy of the
+ *  * License at
+ *  *
+ *  *  http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing,
+ *  * software distributed under the License is distributed on an
+ *  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ *  * either express or implied.  See the License for the specific
+ *  * language governing permissions and limitations under the License.
+ *
+ */
+
+package com.radixdlt.crypto.hdwallet.bitcoinjprovider;
+import com.google.common.base.Joiner;
+import com.google.common.collect.Iterables;
+import com.radixdlt.SecurityCritical;
+import com.radixdlt.crypto.CryptoException;
+import com.radixdlt.crypto.ECKeyPair;
+import com.radixdlt.crypto.hdwallet.HDKeyPair;
+import com.radixdlt.crypto.hdwallet.HDWallet;
+import com.radixdlt.utils.Bytes;
+import org.bitcoinj.crypto.ChildNumber;
+import org.bitcoinj.crypto.DeterministicHierarchy;
+import org.bitcoinj.crypto.DeterministicKey;
+import org.bitcoinj.crypto.HDKeyDerivation;
+import org.bitcoinj.crypto.MnemonicCode;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.bitcoinj.core.Utils.WHITESPACE_SPLITTER;
+
+@SecurityCritical({ SecurityCritical.SecurityKind.KEY_GENERATION, SecurityCritical.SecurityKind.KEY_STORE })
```

Comment:
Why is this flagged KEY_STORE?

Reply:
See my [comment above](https://github.com/radixdlt/radixdlt-java-common/pull/13#discussion_r451412218)